CuratedContent/LanguageReferenceOnLine.md

```diff
@@ -0,0 +1,67 @@
+# Programming Language Reference Manuals On Line
+<!--deck text start-->
+<!--deck text end-->
+
+#### Contributed by [Mark C. Miller](https://github.com/markcmiller86 "Mark C. Miller GitHub Profile")
+
+### C, C++, C Pre-Processor
+
+* [C Preprocessor](https://gcc.gnu.org/onlinedocs/cpp/)
+* [Microsoft C, C++ and Assembler documentation](https://docs.microsoft.com/en-us/cpp/?view=msvc-170)
+* [C++ Reference](https://en.cppreference.com/w/)
+* [C++ Library Reference](https://www.cplusplus.com/reference/)
+* [cplusplus.com](https://www.cplusplus.com)
+* [C Standard Library Reference](https://www.cplusplus.com/reference/clibrary/)
+* [Posix C Library](https://en.wikipedia.org/wiki/C_POSIX_library)
+* [GNU C Reference Manaul](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html)
+* [GNU C Library Reference Manual](https://www.gnu.org/software/libc/manual/pdf/libc.pdf)
+* [C Language Cheat Sheet](https://users.ece.utexas.edu/~adnan/c-refcard.pdf)
+
+### Fortran
+
+* [Fortran 1966-2018 Standards](https://fortranwiki.org/fortran/show/HomePage)
+* [Fortran 90](https://www.fortran90.org)
+* [Fortran 2003](http://www.astrosen.unam.mx/~aceves/Fisica_Computacional/ebooks/handbook_f03.pdf)
```

Comment:
Looks like the original source for this would be a more appropriate link: https://link.springer.com/book/10.1007/978-1-84628-746-6